src/main/java/nextstep/courses/domain/session/Enrollment.java

```diff
@@ -4,20 +4,27 @@ public class Enrollment {
     private final Students students;
     private final int fee;
     private final int maxStudents;
+    private SessionStatus sessionStatus;
 
-    public Enrollment(int fee, int maxStudents) {
+    public Enrollment(int fee, int maxStudents, SessionStatus sessionStatus) {
         this.fee = fee;
         this.maxStudents = maxStudents;
         this.students = new Students();
+        this.sessionStatus = sessionStatus;
```

Comment:
```suggestion
        this(fee, maxStudents, new Students(), sessionStatus);
```
주 생성자를 호출하는 방식으로 구현할 것을 추천

Reply:
생성 로직이 분산되어 있을 필요가 없겠군요. 피드백 감사합니다 :)